### PR TITLE
added problemname and illustraton commands to aspell tex

### LINKED ISSUE #1: Make aspell understand how to work with custom LaTeX commands
WIP

Added this to FKF2024 repo earlier, I find it fist better here as it applies to all future FKF''s.

### PR DESCRIPTION
**Pull request by SvartaHjarta**
Opening this PR early for a discussion, I'm currently looking into more things to improve/customizing this repo for FKF.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ inputs:
   args:
     description: 'Arguments passed to `aspell`.'
     required: false
-    default: '--lang=en_GB --mode=tex'
+    default: '--lang=en_GB --mode=tex --add-tex-command="problemname p" --add-tex-command="illustration ppp"'
```
Comment (Tagl) [inline action.yml]: Kannski bæta href við líka
```suggestion
    default: '--lang=en_GB --mode=tex --add-tex-command="problemname P" --add-tex-command="illustration ppP"'
```

**Comment by SvartaHjarta** [comment]
Perhaps we should ask the others if they feel that there are any other annoyances, or maybe we should just go for another PR?

**Comment by SvartaHjarta** [comment]
Note to self, make sure URL filter is enabled 

And use  `aspell --reset-tex-command config tex-command` to see current tex filters

Should we enable case sensitivity?

**Comment by Tagl** [comment]
I believe case sensitivity is on as there is a difference in behaviour between "Arnar" and "arnar" when adding words to dictionary.

File: entry.rb [hunk 2]
```diff
@@ -11,6 +11,10 @@
 extra_words_files = ARGV[1].shellsplit.flat_map { |path| Dir.glob(path) }
 args = ARGV[2].shellsplit
 
+args << '--add-tex-command=problemname P'
+args << '--add-tex-command=href pP'
+args << '--add-tex-command=illustration ppP'
+
 puts ARGV
```
Comment (SvartaHjarta) [inline entry.rb]:
```suggestion
puts args
```

Probably want to print args as we want the full argument list not just what the user sends in.

**Pull request merged**